.github/workflows/backend-ncnn.yml

```diff
@@ -65,3 +65,25 @@ jobs:
           echo $(pwd)
           ln -s build/bin/mmdeploy_onnx2ncnn ./
           python3 .github/scripts/test_onnx2ncnn.py --run 1
+  script_install:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+        torch: [1.9.0]
+        include:
+          - torch: 1.9.0
+            torch_version: torch1.9
+            torchvision: 0.10.0
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install mmdeploy
+        run: |
+          python3 tools/scripts/build_ubuntu_x64_ncnn.py
```

Comment:
could we add the simple test to check mmdeploy+ncnn is installed correctly? 
maybe `python -c 'import mmdeploy.apis.ncnn as api; assert api.is_available() and api.is_custom_ops_available()'`
and run `python tools/check_env.py` and print the env info.

Reply:
simple test ncnn install added.

cannot execute `check_env.py` because of no mmcv and no pytorch.